benchmarks/routines/gemm.py

```
@@ -1752,6 +1771,192 @@ def run_backend(backend, a, b, bias, use_pdl, out_dtype):
+        )
+    if out_dtype != torch.bfloat16:
+        raise ValueError("tinygemm_bf16 only supports bfloat16 outputs.")
+
+    backends = filter_backends_by_compute_capability(backends, args.routine, device)
+    if len(backends) == 0:
+        print("[ERROR] No backends to test. Exiting.")
+        return res
+
+    from flashinfer.gemm import tinygemm_bf16
+
+    input = torch.randn([m, k], device=device, dtype=torch.bfloat16)
```

Ruff (0.15.10)

[error] Variable `input` is shadowing a Python builtin

(A001)